server/command.go

```diff
@@ -17,10 +17,15 @@ const COMMAND_HELP = `* |/github connect| - Connect your Mattermost account to y
 * |/github todo| - Get a list of unread messages and pull requests awaiting your review
 * |/github subscribe owner/repo [features]| - Subscribe the current channel to receive notifications about opened pull requests and issues for a repository
   * |features| is a comma-delimited list of one or more the following:
-    * issues - includes new issues
-	* pulls - includes new pull requests
+    * issues - includes new and closed issues
+	* pulls - includes new and closed pull requests
+    * pushes - includes pushes
+    * creates - includes branch and tag creations
+    * deletes - includes branch and tag deletions
+    * issue_comments - includes new issue comments
+    * pull_reviews - includes pull request reviews
 	* label:"<labelname>" - must include "pulls" or "issues" in feature list when using a label
-  * Defaults to "pulls,issues"
+  * Defaults to "pulls,issues,pushes,creates,deletes,issue_comments,pull_reviews"
```

Comment:
Can you update the help text too?

Reply:
Right, that's why i shouldn't code in the evening :smile:

Reply:
😅